### PR TITLE
Fix build and tests

### PR DESCRIPTION
**Pull request by GCorbel**
Including `ember-font-awesome` produce : 

    ember-font-awesome: Fonts already imported into the "/fonts" folder [FontAwesome.otf, fontawesome-webfont.eot, fontawesome-webfont.svg, fontawesome-webfont.ttf, fontawesome-webfont.woff, fontawesome-webfont.woff2] by another addon or in your ember-cli-build.js, disable the import from other locations or disable the Font Awesome import by setting `includeFontFiles:false` for the "ember-font-awesome" options in your ember-cli-build.js

It tried to remove all other plugins and clean the ember-cil-build.js and it seems `ember-font-awesome` produce this error when it's the only addon. Using `includeFontFiles: false` fix the problem and the fonts are still included.

### CODE REVIEW

**Comment by spruce** [comment]
I'll try to review today or tomorrow. (I'm kinda stressed out by other things right now.)

**Comment by GCorbel** [comment]
@spruce no problem, don't fill stressed because of this. We do open source, this is not mandatory.

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 # Note: npm is v2.7.6
 RUN npm install -g ember-cli@0.2.7
 RUN npm install -g bower@1.4.1
-RUN npm install -g phantomjs@1.9.16
+
+RUN apt-get install -y libgtk-3-dev \
+  && wget --no-verbose https://ftp.mozilla.org/pub/firefox/releases/56.0/linux-x86_64/en-US/firefox-56.0.tar.bz2 \
+  && tar -xjf firefox-56.0.tar.bz2 \
+  && mv firefox /opt/firefox56 \
+  && ln -s /opt/firefox56/firefox /usr/bin/firefox
```
Comment (spruce) [inline Dockerfile]: Don't we than also need to use firefox with our tests? Currently we only use chrome in ci, no?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 app:
   build: .
-  command: ember s
+  command: ember s --host 0.0.0.0
```
Comment (spruce) [inline docker-compose.yml]: Why is that needed?

File: Dockerfile (hunk 1 above)
Comment (GCorbel) [reply Dockerfile]: Sorry, it wasn't supposed to be there. In the current docker file, we have phantom but it's deprecated. I remove the Dockerfile from this PR.
